fetch/datablobstorage/gcp.go

```
@@ -133,8 +155,21 @@ func listFromContinuationPointGCP(
 			return nil, err
 		} else {
 			if utils.MatchesFileConvention(attrs.Name) {
+				// TODO: need to figure out if this is appropriate behavior or if we should just log and skip?
+				mdNumRows, ok := attrs.Metadata[numRowsKey]
+				if !ok {
+					return nil, err
+				}
+
+				numRows, err := strconv.Atoi(mdNumRows)
+				if err != nil {
+					return nil, err
+				}
```

I think if we run into an error here, its okay to log and swallow it. We don't want to stop because of it. The file itself is probably fine and exists.

---

Ended up logging for both error and missing cases. We continue through the loop in this case though.